charts/vc-authn-oidc/values.yaml

```diff
@@ -700,9 +696,8 @@ postgresql:
     service:
       ports:
         postgresql: 5432
-        ## @param postgresql.primary.extendedConfiguration Extended PostgreSQL Primary configuration (appended to main or default configuration)
-        ## ref: https://github.com/bitnami/containers/tree/main/bitnami/postgresql#allow-settings-to-be-loaded-from-files-other-than-the-default-postgresqlconf
-        ##
-        primary:
-          extendedConfiguration: |
-            max_connections = 500
+    ## @param postgresql.primary.extendedConfiguration Extended PostgreSQL Primary configuration (appended to main or default configuration)
+    ## ref: https://github.com/bitnami/containers/tree/main/bitnami/postgresql#allow-settings-to-be-loaded-from-files-other-than-the-default-postgresqlconf
+    ##
+    extendedConfiguration: |
+      max_connections = 500
```

Comment:
I checked the bitnami repo and the old version seems wrong - this should fix things. See https://github.com/bitnami/charts/tree/main/bitnami/postgresql#postgresql-primary-parameters